src/main/api/2/studio-api-2.yaml

```diff
@@ -3026,81 +3069,108 @@ components:
+              description: The screenshots of the plugin
               items:
-                type: string
-            version:
-              $ref: '#/components/schemas/Version'
-            description:
-              type: string
-              description: The plugin description
-            website:
-              $ref: '#/components/schemas/Link'
-            media:
-              type: object
-              properties:
-                screenshots:
-                  type: array
-                  description: The screenshots of the plugin
-                  items:
-                    $ref: '#/components/schemas/Asset'
-                videos:
-                  type: array
-                  description: The videos of the plugin
-                  items:
-                    $ref: '#/components/schemas/Asset'
-            developer:
-              type: object
-              properties:
-                people:
-                  type: array
-                  description: The people that worked on the plugin
-                  items:
-                    $ref: '#/components/schemas/Link'
-                company:
-                  $ref: '#/components/schemas/Link'
-            build:
-              type: object
-              properties:
-                id:
-                  type: string
-                  description: Build id of the plugin
-                date:
-                  type: string
-                  description: Build date of the plugin
-            license:
-              $ref: '#/components/schemas/Link'
-            crafterCmsVersions:
+                $ref: '#/components/schemas/Asset'
+            videos:
               type: array
-              description: The supported Crafter CMS versions of the plugin
+              description: The videos of the plugin
               items:
-                $ref: '#/components/schemas/Version'
-            crafterCmsEditions:
+                $ref: '#/components/schemas/Asset'
+        developer:
+          type: object
+          properties:
+            people:
               type: array
-              description: The supported Crafter CMS editions of the plugin
+              description: The people that worked on the plugin
               items:
-                type: string
+                $ref: '#/components/schemas/Link'
+            company:
+              $ref: '#/components/schemas/Link'
+        build:
+          type: object
+          properties:
+            id:
+              type: string
+              description: Build id of the plugin
+            date:
+              type: string
+              description: Build date of the plugin
+        license:
+          $ref: '#/components/schemas/Link'
+        crafterCmsVersions:
+          type: array
+          description: The supported Crafter CMS versions of the plugin
+          items:
+            $ref: '#/components/schemas/Version'
+        crafterCmsEditions:
```

Comment:
Should we be returning this to the UI? Or just return Community to Community, and everything to E?
Should we disallow install of E on C in the backend regardless of what the UI lists?
Should the UI explain that E won't work with C?

I feel it best to simply not list E in C, and list all in E.

Let's discuss what's feasible.

Reply:
I agree, we can return only the ones that are compatible, same for the `crafterCmsVersions`. Should we remove the fields from the result to avoid confusion?

Reply:
Perhaps the UI should indicate if something is E to the E customers. C customer won't be bothered because all returned is C.